Source/DatabaseManager/Appsettings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "ConnectionStrings": {
+    "DefaultConnection": "SERVER = den1.mssql8.gear.host; DATABASE = theludogame; UID = theludogame; PWD = Td7p_l8gB~5T"

```

Comment:
Watch out putting connection strings to servers in a public git repo
You should add this file to your gitignore and change the password asap!